### ISSUE TITLE
A11Y: Mobile nav is not keyboard accessible

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/clean-and-green-philly#728]
## Describe the task

Note: This ticket may be a big pain. Just FYI before you grab it.

When the screen is narrower and the mobile nav hamburger button pops up, it is not reachable or operable via keyboard navigation.

You should be able to TAB to the hamburger menu, press ENTER or SPACE and the dropdown opens. Then you can use the UP or DOWN arrow keys to navigate the items (with focus state visible). Press ENTER or SPACE to choose one of the items. 

While the dropdown is open pressing ESC or TAB will close the dropdown and focus will go back to the hamburger button. 

See this example for most of the desired behavior (except for the behavior while the dropdown is open and pressing TAB.) https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation/#mythical-page-content

**Comment by nlebovits** [comment]
@bacitracin, @CodeWritingCow and I are wondering if this can be broken into smaller tickets so that it's a little less intimidating for volunteers

**Comment by bacitracin** [comment]
@nlebovits @CodeWritingCow Yeah I can try. Pulling this to backlog while I rewrite

**Comment by bacitracin** [comment]
Broke this ticket down into 3 tickets that should be completed in order.
https://github.com/CodeForPhilly/clean-and-green-philly/issues/978
https://github.com/CodeForPhilly/clean-and-green-philly/issues/979
https://github.com/CodeForPhilly/clean-and-green-philly/issues/980
